source/documentation/doc-Release_Notes/topics/ref-ovirt-4.5.2.adoc

```diff
@@ -0,0 +1,102 @@
+If your ovirt-provider-ovn certificate expires on a previous RHV version, please upgrade to RHV 4.4 SP1 batch 2 or newer, and ovirt-provider-ovn certificate will be renewed automatically as a part of engine-setup.
+
+BZ#link:https://bugzilla.redhat.com/2104939[2104939]::
+With this release, OVA export or import work on hosts with a non-standard SSH port.
+
+BZ#link:https://bugzilla.redhat.com/2107250[2107250]::
+With this release, the process to check certificate validity is now compatible with both RHEL 8 and RHEL 7 based hypervisors.
+
+==== Rebase: Bug Fixes and Enhancements
+
+These items are rebases of bug fixes and enhancements included in this release of Red Hat Virtualization:
+
+BZ#link:https://bugzilla.redhat.com/2092478[2092478]::
+UnboundID LDAP SDK has been rebased on upstream version 6.0.4. Please review https://github.com/pingidentity/ldapsdk/releases for changes since version 4.0.14
+
+==== Rebase: Bug Fixes Only
+
+These items are rebases of bug fixes included in this release of Red Hat Virtualization:
+
+BZ#link:https://bugzilla.redhat.com/2104831[2104831]::
+Rebase package(s) to version: 4.4.7.
+Highlights, important fixes, or notable enhancements: fixed BZ#2081676
+
+==== Release Notes
+
+This section outlines important details about the release, including recommended practices and notable changes to Red Hat Virtualization. You must take this information into account to ensure the best possible outcomes for your deployment.
+
+BZ#link:https://bugzilla.redhat.com/2049286[2049286]::
+With this release, only virtual machines pinned to hosts and selected for upgrade during cluster upgrade are stopped during the cluster upgrade. VMs pinned to other hosts (not selected for upgrade) are left untouched.
+
+BZ#link:https://bugzilla.redhat.com/2108985[2108985]::
+RHV 4.4 SP1 and later are only supported on RHEL 8.6, so customers cannot use RHEL 8.7 or later, and must stay with RHEL 8.6 EUS.
+
+BZ#link:https://bugzilla.redhat.com/2113068[2113068]::
+With this release, permissions for the /var/log/ovn directory are updated correctly during the upgrade of OVS/OVN 2.11 to OVS 2.15/OVN 2021.
+
+==== Deprecated Functionality
+
+The items in this section are either no longer supported, or will no longer be supported in a future release.
+
+BZ#link:https://bugzilla.redhat.com/2111600[2111600]::
+ovirt-engine-extension-aaa-jdbc and ovirt-engine-extension-aaa-ldap are deprecated in RHV 4.4 SP1, they are going to remain within RHV product, but for any new request customer should use integration with RH Single Sign On as mentioned in https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.4/html/administration_guide/configuring_red_hat_sso
```

Comment:
The English needs to be fixed. Please change the link so that it follows our style conventions: link:url[text]. Fix the comma splice. We don't normally say "customer". Usually we say "you".